src/coffee/compress.coffee

```diff
@@ -26,11 +26,11 @@ class Compress
             .use(plugin)
             .run (err) ->
               if err
-                resolve 'Cannot compress file: %s', err
+                reject 'error while compressing file: ' + err
```

Comment:
Please always start sentence with upper case. Search for most tools is case insensitive